src/Pusher.php

```diff
@@ -131,9 +77,9 @@ public function __construct($auth_key, $secret, $app_id, $options = array(), $ho
         /* End backward compatibility with old constructor **/
 
         if (isset($options['encrypted']) &&
-                $options['encrypted'] === true &&
-                !isset($options['scheme']) &&
-                !isset($options['port'])) {
+        $options['encrypted'] === true &&
```

Comment:
I'm unsure about this stylistic change as then this reads almost as a separate instruction and not a continuation of the if statement.